### DISCUSSION TITLE
Feature: Optionally disable application mode (indexing and searching)

### DISCUSSION THREAD

**Discussion by v6cc** [Ulauncher/Ulauncher#901] [Ideas]
I want to change from `albert`(https://github.com/albertlauncher/albert) to `ulauncher` , a problem is I never use it open application , only use it search web

In albert, I can close the applicitions extensions,   not show them 

Could you please also support close this in ulauncher?

**Comment by troycurtisjr** [comment]
@zdmgmail are you asking to be able to close running applications, instead of just being able to open them? If so checkout https://github.com/Ulauncher/ulauncher-kill . If not, could you give a little more detail about what you are looking for?

**Comment by friday** [comment]
I should have paid more attention to the description. I think I realize what this means now. The author wants to toggle ulauncher, enter something, and not be shown a list of apps matching his input. He wants the shortcuts like "Search on Wikipedia" etc to be the only things shown.

I'm note sure it's something we want to support, but it wouldn't technically be that hard.

**Comment by troycurtisjr** [comment]
Basically taking the "launcher" out of "Ulauncher"? 

I wonder if you could effectively do this by blacklisting the main applications directory in the Ulauncher config?
